src/views/Home.vue

```diff
@@ -22,5 +22,10 @@ export default {
     Title,
     Button,
   },
+  methods: {
+    firstPage(){
```

Comment:
Consider naming methods with a verb to indicate that they are an action:

e.g. `goToFirstPage` would be clearer that it is an function/action/method at a glance